doc/manual/queryformat

```diff
@@ -153,10 +154,10 @@ braces.
 
 \section queryformat_example Example: Viewing the Verify Flags
 
-The following example query is run against dev because I know %verify
-is used there.
+The following example query is run against nss because %verify is used
+there.
 \verbatim
-	rpm -q --qf '[%{filenames} %{fileverifyflags}\n]' dev
+	rpm -q --qf '[%{FILENAMES} %{FILEVERIFYFLAGS:hex}\n]' nss
```

Comment:
To really improve this example, I'd suggest using something else than FILEVERIFYFLAGS for the data. FILEMODES:octal might be a better example as all packages with files have them, and the values are commonly known, so you could delete the rpmfiles.h reference and values from the document too.